examples/python/open_photogrammetry_format/main.py

```diff
@@ -0,0 +1,229 @@
+        pcl = self.project.point_cloud_objs[0]
+        rr.log_points("world/pcl", positions=pcl.nodes[0].position, colors=pcl.nodes[0].color, timeless=True)
+
+    def log_calibrated_cameras(self) -> None:
+        """
+        Log the project's calibrated cameras as individual frames.
+
+        Logging all cameras in a single frame is also possible, but clutter the default view with too many image views.
+        """
+        sensor_map = {sensor.id: sensor for sensor in self.project.input_cameras.sensors}
+        calib_sensor_map = {sensor.id: sensor for sensor in self.project.calibration.calibrated_cameras.sensors}
+
+        for i, (camera, calib_camera) in enumerate(
+            zip(
+                self.project.camera_list.cameras,
+                self.project.calibration.calibrated_cameras.cameras,
+            )
+        ):
+            if not str(camera.uri).endswith(".jpg"):
+                continue
+
+            if self.log_as_frames:
+                rr.set_time_sequence("image", i)
+                entity = "world/cameras"
+            else:
+                entity = f"world/cameras/{i}"
+
+            sensor = sensor_map[calib_camera.sensor_id]
+            calib_sensor = calib_sensor_map[calib_camera.sensor_id]
+
+            omega, phi, kappa = tuple(np.deg2rad(a) for a in calib_camera.orientation_deg)
+            rot = (
+                np.array(
+                    [
+                        [1, 0, 0],
+                        [0, np.cos(omega), -np.sin(omega)],
+                        [0, np.sin(omega), np.cos(omega)],
+                    ]
+                )
+                @ np.array(
+                    [
+                        [np.cos(phi), 0, np.sin(phi)],
+                        [0, 1, 0],
+                        [-np.sin(phi), 0, np.cos(phi)],
+                    ]
+                )
+                @ np.array(
+                    [
+                        [np.cos(kappa), -np.sin(kappa), 0],
+                        [np.sin(kappa), np.cos(kappa), 0],
+                        [0, 0, 1],
+                    ]
+                )
+                @ np.array(  # somehow needed to have the camera point at the right direction
+                    [
+                        [1, 0, 0],
+                        [0, -1, 0],
+                        [0, 0, -1],
+                    ]
+                )
+            )
+
+            rr.log_transform3d(entity, rr.TranslationAndMat3(translation=calib_camera.position, matrix=rot))
+
+            assert calib_sensor.internals.type == "perspective"
+
+            focal_length = calib_sensor.internals.focal_length_px
+            u_center = calib_sensor.internals.principal_point_px[0]
+            v_center = calib_sensor.internals.principal_point_px[1]
+            intrinsics = np.array(
+                (
+                    (focal_length, 0, u_center),
+                    (0, focal_length, v_center),
+                    (0, 0, 1),
+                )
```

Comment:
It feels weird to me to use tuples here instead of arrays, and inconsistent with earlier code
```suggestion
                [
                    [focal_length, 0, u_center],
                    [0, focal_length, v_center],
                    [0, 0, 1],
                ]
```